aten/src/ATen/native/special_functions/detail/complex.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
```

Comment:
@albanD polyfills `std::isnan` and `std::isinf` for `c10::complex`. This could be moved to c10 in the future (or removed once we move to 17).